backend/src/ee/services/license/license-service.ts

```diff
@@ -222,13 +223,67 @@ export const licenseServiceFactory = ({
     const {
       data: { url }
     } = await licenseServerCloudApi.request.post(
-      `/api/license-server/v1/customers/${organization.customerId}/session/trail`,
+      `/api/license-server/v1/customers/${organization.customerId}/session/trial`,
       { success_url }
     );
     featureStore.del(FEATURE_CACHE_KEY(orgId));
     return { url };
   };
 
+  const createOrganizationPortalSession = async ({ orgId, actorId, actor }: TCreateOrgPortalSession) => {
+    const { permission } = await permissionService.getOrgPermission(actor, actorId, orgId);
+    ForbiddenError.from(permission).throwUnlessCan(
+      OrgPermissionActions.Create,
+      OrgPermissionSubjects.Billing
+    );
+    ForbiddenError.from(permission).throwUnlessCan(
+      OrgPermissionActions.Edit,
+      OrgPermissionSubjects.Billing
+    );
+
+    const organization = await orgDAL.findOrgById(orgId);
+    if (!organization) {
+      throw new BadRequestError({
+        message: "Failed to find organization"
+      });
+    }
+
+    const {
+      data: { pmtMethods }
+    } = await licenseServerCloudApi.request.get(
```

Comment:
One comment would be it would be great for all external request we set the type instead of any. Eslint will catch this going forward.

That is request.get<{pmtMethods: <type>}>()